module/migrations/PrimeMigration_0_3_1_to_0_4_0.js

```diff
@@ -0,0 +1,28 @@
+export class PrimeMigration_0_3_1_to_0_4_0
```

Comment:
I take it this is just a placeholder for the data munging logic we're likely going to need to add?

Reply:
Yup - probably pint some calls to migrate v1 actors to v2 inside here. I'm working on some bits with that right now, probably do to have a chat about the final shape of things but I'm just making up some tools at the moment.